packages/magento-product/components/ProductPagePrice/ProductPagePriceTiers.tsx

```diff
@@ -0,0 +1,31 @@
+import { Money } from '@graphcommerce/magento-store';
+import { Trans } from '@lingui/react';
+import { Box, SxProps, Theme } from '@mui/material';
+import { ProductPagePriceFragment } from './ProductPagePrice.gql';
+
+type ProductPagePriceTiersProps = {
+  sx?: SxProps<Theme>
+} & ProductPagePriceFragment
+
+export function ProductPagePriceTiers(props: ProductPagePriceTiersProps) {
+  const { price_tiers, sx = [] } = props;
+
+  return (
```

Comment:
This will always output a div, even if there aren't any price tiers. You can see the layout jump here: https://graphcommerce-git-fork-haelbichalex-featur-a99009-graphcommerce.vercel.app/nl/p/tamara-socks-gc-1038-sock

Reply:
Fixed ✅